persist/src/main/resources/db/migration/V1__Initial_Import copy.sql

```diff
@@ -70,4 +70,18 @@ IF NOT EXISTS blacklisted_tokens
   id VARCHAR
 (255) NOT NULL PRIMARY KEY,
   time_milliseconds BIGINT
+);
```

Comment:
**Don't edit the migration files!** You need to create a new migration file or flyway is not going to pick up this change